knope.toml

```diff
@@ -83,6 +83,9 @@ versioned_files = ["crates/biome_css_parser/Cargo.toml"]
 [packages.biome_css_syntax]
 changelog       = "crates/biome_css_syntax/CHANGELOG.md"
 versioned_files = ["crates/biome_css_syntax/Cargo.toml"]
+[packages.biome_css_analyze]
```

Comment:
![CleanShot 2025-02-02 at 19 55 06](https://github.com/user-attachments/assets/ff854c31-55f2-41a4-8993-13f0e6fcf223)

I found that `biome_css_analyze` is missing from the knope's configuration file. Would you mind checking if I did this correctly by adding it here manually? @ematipico

Reply:
That's correct, thank you!